python/run-tests

```diff
@@ -96,6 +98,21 @@ function run_ml_tests() {
 
 function run_streaming_tests() {
     echo "Run streaming tests ..."
+
+    KAFKA_ASSEMBLY_DIR="$FWDIR"/external/kafka-assembly
+    JAR_PATH="${KAFKA_ASSEMBLY_DIR}/target/scala-${SPARK_SCALA_VERSION}"
+    for f in "${JAR_PATH}"/spark-streaming-kafka-assembly-*.jar; do
+      if [[ ! -e "$f" ]]; then
+        echo "Failed to find Spark Streaming Kafka assembly jar in $KAFKA_ASSEMBLY_DIR" 1>&2
+        echo "You need to build Spark with " \
+             "'build/sbt assembly/assembly streaming-kafka-assembly/assembly' or" \
+             "'build/mvn package' before running this program" 1>&2
+        exit 1
```

Comment:
I guess this means that we'll bail out of the rest of the Python tests if the Kafka assembly hasn't been built; I suppose that we could print a failure message instead and try to continue, but that might not be worth the extra effort right now: you generally have to build an assembly before running Python tests and I don't know that it's worth optimizing for the case where someone has gone out of their way to skip the Kafka assembly.